index.js

```diff
@@ -61,6 +62,11 @@ function onModelChange(e) {
     toItem.classList.add(this._options.activeDescendantClassName);
 
     this._focusEl.setAttribute('aria-activedescendant', toItem.id);
+
+    if (this._config.scrollInView && this._containerEl) {
+      this._containerEl.scrollTop = toItem.offsetTop - this._containerEl.offsetHeight / 2;
```

Comment:
I do wonder whether we should just include the `toItem.offsetTop` data in the activeDescendantChange event, to let an observer handle the scroll behaviour. But as this is still an experimental module, and we need this feature quickly, I'm happy to go with it.  Maybe we should rename it to `autoScroll` though?